l10n/de_DE.json

```diff
@@ -169,6 +169,7 @@
     "Password request: %s" : "Passwortanforderung: %s",
     "Private conversation" : "Private Unterhaltung",
     "Deleted user (%s)" : "Gelöschter Benutzer (%s)",
+    "{user1} shared room {roomName} on {remoteServer} with you" : "{user1} hat den Raum {roomName} auf {remoteServer} mit Ihnen geteilt",
```

Comment:
something went wrong with rebase with master ?

Reply:
Maybe that happened when I tried to fix my missing signoffs with the recommended command via copy&paste :-S